packs/fireeye/actions/submit_malware.py

```diff
@@ -0,0 +1,16 @@
+from lib.actions import BaseAction
+import json
+import os
+
+
+class SubmitMalwareAction(BaseAction):
+    def run(self, **kwargs):
```

Comment:
Looking at the action metadata file, it also defines `file_path` and `file_type` parameters.

I assume we don't want to send those in the payload? Or it doesn't matter and it still works even if those parameters are included?